From 373870298e39f1ce4dedb6a87e3a24925cd88644 Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Mon, 1 Sep 2014 09:25:28 -0400
Subject: [md] Convert use of typedef ctl_table to struct ctl_table

Message-id: <1409563584-9000-27-git-send-email-Jes.Sorensen@redhat.com>
Patchwork-id: 89964
O-Subject: [RHEL7.1 PATCH 26/82] md: Convert use of typedef ctl_table to struct ctl_table
Bugzilla: 1085530
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

From: Joe Perches <joe@perches.com>

This typedef is unnecessary and should just be removed.

Signed-off-by: Joe Perches <joe@perches.com>
Signed-off-by: NeilBrown <neilb@suse.de>
(cherry picked from commit 82592c38a85889fc9b52bf67afd4f6a336858a96)
Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/md/md.c b/drivers/md/md.c
index f8c56ab..5f1a5ce 100644
--- a/drivers/md/md.c
+++ b/drivers/md/md.c
@@ -112,7 +112,7 @@ static inline int speed_max(struct mddev *mddev)
 
 static struct ctl_table_header *raid_table_header;
 
-static ctl_table raid_table[] = {
+static struct ctl_table raid_table[] = {
  {
   .procname = "speed_limit_min",
   .data  = &sysctl_speed_limit_min,
@@ -130,7 +130,7 @@ static ctl_table raid_table[] = {
  { }
 };
 
-static ctl_table raid_dir_table[] = {
+static struct ctl_table raid_dir_table[] = {
  {
   .procname = "raid",
   .maxlen  = 0,
@@ -140,7 +140,7 @@ static ctl_table raid_dir_table[] = {
  { }
 };
 
-static ctl_table raid_root_table[] = {
+static struct ctl_table raid_root_table[] = {
  {
   .procname = "dev",
   .maxlen  = 0,
-- 
1.7.1